### ISSUE TITLE
HTML5 tags <details> wrapped by <p> tag

### ISSUE THREAD

**Issue by chrispcb** [tinymce/tinymce#3276]
hello,

Tested on your home page:
- switch to source code (tools ->source code)
- add the end, add this code:

```
<details>
<summary>Titre details</summary>
<p>hidden texte</p>
</details>
```
- click "ok"
- switch to source code again (tools ->source code)
- Wrong Result:

```
<p>&nbsp;</p>
<p><details> <summary>test details</summary>
<p>test</p>
</details></p>
<p>&nbsp;</p>
```

**Comment by bastien** [comment]
Would love to see this fixed 👍

**Comment by spocke** [comment]
Yeah, I guess we should treat details as a block element not wrapping it in a paragraph.

**Comment by sunesimonsen** [comment]
Any new on this issue?

**Comment by spocke** [comment]
We added a fix for this in 4.7.12 so you could try that out. These elements are now handled as block element containers. We don't have any special editing on them so they are just retained as is at the moment.

**Comment by chrispcb** [comment]
It's ok. Good job !
